fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/RuleSet.java

```diff
@@ -512,7 +519,9 @@ public void addCboTablePruneRule() {
     }
 
     public void addMultiTableMvRewriteRule() {
-        transformRules.addAll(REWRITE_RULES.get(RuleSetType.MULTI_TABLE_MV_REWRITE));
+        transformRules.add(AggregateJoinRule.getInstance());
```

Comment:
why not use `MULTI_TABLE_MV_REWRITE_RULES`?

Reply:
emmm, memo wasn't support `predecessorRules` in this PR, it's need more test, I will support it later